packages/components/cascader-panel/src/instances.ts

```diff
@@ -0,0 +1,3 @@
+import type CascaderPanel from './index.vue'
```

Comment:
## Suggest

<img width="251" alt="image" src="https://user-images.githubusercontent.com/27342882/222418929-f91b7beb-7dd0-4593-9578-5ddce18bf78e.png">

there are two kinds of instance file names. I think it would be nice to unify them into one! (`instance.ts`)

Reply:
You'r right.